src/main/java/org/apache/sling/distribution/journal/ExceptionEventSender.java

```diff
@@ -46,6 +47,12 @@ private Event createErrorEvent(Exception e) {
         Map<String, String> props = new HashMap<>();
         props.put(KEY_TYPE, e.getClass().getName());
         props.put(KEY_MESSAGE, e.getMessage());
+        if (e instanceof MessagingException) {
```

Comment:
One small suggestion, instead of using instanceof, we may override the method `createErrorEvent` with a method that passes a `MessagingException` as argument.

Reply:
Yes thanks but would also need to override the `send` method.

Reply:
Good point, and bump the package export from 0.1.0 to 0.2.0.

Reply:
Let's track that improvement separately.